hackathon_site/dashboard/frontend/src/components/general/TitledPaper/__snapshots__/TitledPaper.test.js.snap

```diff
@@ -0,0 +1,16 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
As above, I don't think this is a particularly helpful snapshot to have, but I defer to your judgement.